common/test/acceptance/tests/lms/test_certificate_web_view.py

```diff
@@ -218,11 +218,11 @@ def complete_course_problems(self):
         self.course_nav.q(css='select option[value="{}"]'.format('blue')).first.click()
 
         # Select correct radio button for the answer
-        self.course_nav.q(css='fieldset label:nth-child(3) input').nth(0).click()
+        self.course_nav.q(css='fieldset div.field:nth-child(4) input').nth(0).click()
```

Comment:
Nit: we generally don't put specific tag types in the selectors. Using the class name of "field" should be enough. Same note applies in the file below, at lines 866-868.